mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -41,29 +47,79 @@ def __copy__(self):
         return result
 
     def get_storage_write_cache(self, iteration: int):
-        if iteration not in self.storage_written:
-            self.storage_written[iteration] = []
-
-        return self.storage_written[iteration]
+        return self.storage_written.get(iteration, set())
 
     def extend_storage_write_cache(self, iteration: int, value: object):
-        if iteration not in self.storage_written:
-            self.storage_written[iteration] = [value]
-        elif value not in self.storage_written[iteration]:
-            self.storage_written[iteration].append(value)
-
-
-class WSDependencyAnnotation(StateAnnotation):
+        self.storage_written.get(iteration, set()).add(value)
```

Comment:
This doesn't work when the default value is passed
![image](https://user-images.githubusercontent.com/8710366/95836817-2da83b00-0d40-11eb-8d02-c03d1794e3b4.png)

Reply:
😱